### PR TITLE
WIP: PB tools

### PR DESCRIPTION
**Pull request by aawdls**
For review

### CODE REVIEW

File: aa/pb_tools/cli.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+@click.argument("input_path", type=click.Path(exists=True))
+@click.help_option('--help', '-h')
+def report_pb_file_errors(input_path):
+    """
+    Check for known types of errors in the protocol buffer
+    file at INPUT_PATH. Returns 0 if there are no errors,
+    otherwise prints a list of errors to standard out
+    and returns 1.
+    """
+    file = pb_file.PbFile(input_path)
```
Comment (willrogers) [inline aa/pb_tools/cli.py]: `file` isn't a great variable name as it's a builtin function

File: aa/storage.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import re
+
+STORAGE_PATHS = {"LTS": "/dls/science/users/tdq39642/aa/mock_lts"}
```
Comment (willrogers) [inline aa/storage.py]: Your path left in?

**Comment by willrogers** [comment]
This looks nice and generally useful, thanks.

There is the question about an optional dependency on PyQt5. Also, now we have `pb_tools` perhaps we should move the PB utility code into there. But other than that I don't have many comments, looks good to me.

**Comment by aawdls** [comment]
Have now moved this stuff to a separate module. Making anew PR with just the required changes to aapy.

**Pull request closed without merging**